client/datatypes.go

```diff
@@ -14,6 +14,12 @@ package client
 
 import "net/http"
 
+// Field user type.
+type Field struct {
```

Comment:
I am doing simple, crap, crud support in this sprint. I have seen the referenced doc and I am aiming for something similar in the end. Not in this PR, though.